spring-cloud-task-core/src/main/resources/org/springframework/cloud/task/schema-h2.sql

```diff
@@ -16,4 +16,11 @@ CREATE TABLE TASK_EXECUTION_PARAMS  (
 	references TASK_EXECUTION(TASK_EXECUTION_ID)
 ) ;
 
+CREATE TABLE TASK_TASK_BATCH (
+  TASK_EXECUTION_ID BIGINT NOT NULL ,
+  JOB_EXECUTION_ID BIGINT NOT NULL ,
+	constraint TASK_EXEC_BATCH_FK foreign key (TASK_EXECUTION_ID)
+	references TASK_EXECUTION(TASK_EXECUTION_ID)
+) ;
+
```

Comment:
Need to include other DB Schemas (I'm such a nag). For this PR we can add the MySQL.  For the others we can add a story and I can add those.